RedHat-UX/red-hat-design-system
# feat: update UX dot's table styling

## What I did

Change UX dot tables to look like `<rh-table>`

## Testing Instructions

Check that tables throughout the DP look similar. Some examples are on the [Spacing](https://deploy-preview-1412--red-hat-design-system.netlify.app/foundations/spacing/), [Accordion's Guidelines](https://deploy-preview-1412--red-hat-design-system.netlify.app/elements/accordion/guidelines/), and [Release Notes](https://deploy-preview-1412--red-hat-design-system.netlify.app/release-notes/) pages.


## Notes to Reviewers
